omc3/correction/response_twiss.py

```diff
@@ -741,7 +741,7 @@ def disp_caller(func, plane):
 
         def tune_caller(func, _unused):
             tune = func()
-            res = tune["X"].append(tune["Y"])
+            res = pd.concat([tune["X"], tune["Y"]], axis="index", ignore_index=True)
```

Comment:
Are these pandas DataFrames or `TfsDataFrames`? Or if we don't care about indexes here, then sure.

Reply:
we don't care about the header here, so pd is fine